tests/t0_http.py

```diff
@@ -483,34 +483,11 @@ def test_discard_chunked(conn, monkeypatch):
 def test_read_text(conn):
     conn.send_request('GET', '/send_%d_bytes' % len(DUMMY_DATA))
     conn.read_response()
-    fh = TextIOWrapper(conn)
+    fh = TextIOWrapper(BytesIO(conn.readall()))
     assert fh.read() == DUMMY_DATA.decode('utf8')
     assert not conn.response_pending()
 
 
-def test_read_text2(conn):
```

Comment:
Why are we removing all these?

Reply:
These tests look like they are testing compatibility with TextIOWrapper. That was a [dugong](https://pypi.org/project/dugong/) feature that S3QL does not need.

Reply:
As I said above, maybe we could just implement flush (as no-op) and close (equivalent to disconnect)? I feel like this might be useful in the future.